test/generic/MPoly-test.jl

```diff
@@ -92,6 +92,21 @@
       @test !(y in keys(Dict(x => 1)))
    end
 
+   R, (x, y) = QQ["x", "y"]
```

Comment:
This needs another letter or the `R` in line 96 needs to be changed.

Reply:
Thanks.